lib/savings/view/savings_screen.dart

```diff
@@ -0,0 +1,100 @@
+// ignore_for_file: omit_local_variable_types, prefer_final_locals
+
+import 'package:flutter/material.dart';
+import 'package:fpb/assets/fpb_svg.dart';
+import 'package:fpb/l10n/l10n.dart';
+import 'package:fpb/savings/view/widgets/account_card.dart';
+import 'package:fpb/savings/view/widgets/account_type_tile.dart';
+import 'package:fpb/savings/view/widgets/savings_screen_title.dart';
+
+class SavingsPage extends StatefulWidget {
+  const SavingsPage({super.key});
+  static const String routeName = '/savings';
```

Comment:
Please make sure to add this page to the `HomeScreen` in the `app_route.dart` file

Reply:
Okay